db/migrate/20200805121536_change_submissions_to_reports.rb

```diff
@@ -0,0 +1,16 @@
+class ChangeSubmissionsToReports < ActiveRecord::Migration[5.0]
+  def self.up
+    remove_reference :transactions, :submission
+    add_reference :transactions, :report, type: :uuid
+  end
+
+  def self.down
+    remove_reference :transactions, :report
+    add_reference :transactions, :submission, type: :uuid
+  end
+
+  def change
```

Comment:
Also, if you remove the reference and add it again, it'll wipe out the values and you'll lose the existing assoications.

I'm not sure if you can mix `up/down` with `change` - try removing `up` and `down` and using:

```ruby
def change
  rename_table :submissions, :reports
  rename_column :transactions, :submission_id, :report_id
  remove_index :transactions, :submission_id
  add_index :transactions, report_id
end

Reply:
Ah! Nice, thank you! It seems that renaming the column also does the index, so clean, ❤️  Rails! 😄